package.json

```diff
@@ -5,6 +5,7 @@
   "main": "app.js",
   "scripts": {
     "start": "nodemon app.js",
+    // AKOSREVIEW - I think this and dev should switch
     "dev": "node app.js "
```

Comment:
Usually `dev` script is for running things locally, so we'd want `nodemon` there which automatically restarts the server anytime something changes. `start` is usually used for when we actually deploy code to like Render.com, where we want the server to just run indefinitely without restarting if possible